Code/Editor/MainWindow.cpp

```diff
@@ -116,16 +116,18 @@ using namespace AzToolsFramework;
 #define LAYOUTS_WILDCARD "*.layout"
 #define DUMMY_LAYOUT_NAME "Dummy_Layout"
 
-static constexpr const char* EditorMainWindowActionContextIdentifier = "o3de.context.editor.mainwindow";
+static constexpr AZStd::string_view EditorMainWindowActionContextIdentifier = "o3de.context.editor.mainwindow";
 
-static constexpr const char* FileMenuIdentifier = "o3de.menu.editor.file";
-static constexpr const char* EditMenuIdentifier = "o3de.menu.editor.edit";
-static constexpr const char* GameMenuIdentifier = "o3de.menu.editor.game";
-static constexpr const char* ToolsMenuIdentifier = "o3de.menu.editor.tools";
-static constexpr const char* ViewMenuIdentifier = "o3de.menu.editor.view";
-static constexpr const char* HelpMenuIdentifier = "o3de.menu.editor.help";
-static constexpr const char* HelpDocumentationMenuIdentifier = "o3de.menu.editor.help.documentation";
-static constexpr const char* HelpGameDevResourcesMenuIdentifier = "o3de.menu.editor.help.gamedevresources";
+static constexpr AZStd::string_view EditorMainWindowMenuBarIdentifier = "o3de.menubar.editor.mainwindow";
+
+static constexpr AZStd::string_view FileMenuIdentifier = "o3de.menu.editor.file";
+static constexpr AZStd::string_view EditMenuIdentifier = "o3de.menu.editor.edit";
+static constexpr AZStd::string_view GameMenuIdentifier = "o3de.menu.editor.game";
+static constexpr AZStd::string_view ToolsMenuIdentifier = "o3de.menu.editor.tools";
+static constexpr AZStd::string_view ViewMenuIdentifier = "o3de.menu.editor.view";
+static constexpr AZStd::string_view HelpMenuIdentifier = "o3de.menu.editor.help";
+static constexpr AZStd::string_view HelpDocumentationMenuIdentifier = "o3de.menu.editor.help.documentation";
+static constexpr AZStd::string_view HelpGameDevResourcesMenuIdentifier = "o3de.menu.editor.help.gamedevresources";
```

Comment:
question: This is probably fine, but we have to be very careful we don't end up using these with any C-style APIs expecting a null terminated string

Reply:
Yes. These are meant to be used with the ActionManager API that accepts const AZStd::string, so it made sense to change them as it speeds things up a bit. I'll keep that in mind though :D

Reply:
Perfect thanks! 🙂